lib/fuzzy-finder-view.coffee

```diff
@@ -61,9 +63,12 @@ class FuzzyFinderView extends SelectListView
         @div fileBasename, class: "primary-line file icon #{typeClass}", 'data-name': fileBasename, 'data-path': projectRelativePath
         @div projectRelativePath, class: 'secondary-line path no-icon'
 
-  openPath: (filePath, lineNumber) ->
+  openPath: (filePath, lineNumber, searchAllPanes) ->
     if filePath
-      atom.workspace.open(filePath).done => @moveToLine(lineNumber)
+      if atom.workspace.getActivePane().activateItemForURI(filePath)
+        @moveToLine(lineNumber)
+      else
+        atom.workspace.open(filePath, searchAllPanes: searchAllPanes ).done => @moveToLine(lineNumber)
```

Comment:
I swear there was a reason, but it works and tests just fine simplified. Changed.